client/src/modules/inventory/list/modals/search.modal.js

```diff
@@ -33,6 +37,14 @@ function InventorySearchModalController(ModalInstance, Notify, filters, Inventor
   vm.defaultQueries = {};
   vm.searchQueries = {};
 
+  var lastViewFilters = Inventory.filters.formatView().customFilters;
+
+  // map key to last display value for lookup in loggedChange
+  var lastDisplayValues = lastViewFilters.reduce(function (object, filter) {
```

Comment:
Since we are doing this so much, we could probably bake it into the `FilterService`.  Something like:
```js
var lastDisplayValues = Inventory.filters.getDisplayMap();
```
This would be a separate issue and PR.